tests/test_models.py

```diff
@@ -24,6 +24,7 @@
 from service import app
 
 DATABASE_URI = os.getenv("DATABASE_URI", "sqlite:///../db/test.db")
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
```

Comment:
Why have we set it to False? Just curious, I don't think it's a bug or something but wanted to know why we do this

Reply:
This is a historical issue. SQLAlchemy will print a lot of warning if we did not set it to False. 

We will create a config file later and move this line there.